test/unit/actions.test.js

```diff
@@ -60,7 +79,7 @@ test('should retrieve action from string identifier', t => {
     t.is(path, `namespaces/${ns}/actions/12345`)
   }
 
-  return actions.get('12345')
```

Comment:
Did you mean to make this change?

Reply:
@jthomas, yes this change was needed to prevent an error now that `action.get` doesn't use the default `get` method from `resources.js`.

Reply:
@dubeejw The test case was throwing an error because this change [broke that API](https://github.com/apache/incubator-openwhisk-client-js#invoke-action). Can you submit a PR to revert this test case change and modify the `get` method to handle the string identifier. There's a `parse_options` method that you can use to handle the string case correctly. https://github.com/apache/incubator-openwhisk-client-js/blob/master/lib/resources.js#L64-L70

We'll push this out as a patch release.